stable/rabbitmq-ha/README.md

```diff
@@ -142,6 +142,7 @@ and their default values.
 | `rabbitmqWebMQTTPlugin.enabled`                | Enable MQTT over websocket plugin                                                                                                                                                                     | `false`                                                    |
 | `rabbitmqWebSTOMPPlugin.config`                | STOMP over websocket configuration                                                                                                                                                                    | ``                                                         |
 | `rabbitmqWebSTOMPPlugin.enabled`               | Enable STOMP over websocket plugin                                                                                                                                                                    | `false`                                                    |
+| `rabbitmqPrometheus.enabled`               | Enable native RabbitMQ prometheus plugin                                                                                                                                                                    | `false`                                                    |
```

Comment:
Add a note to the prometheus section at the bottom in readme about if you're using RabbitMQ 3.8 you should not use `prometheus.exporter.enabled=true` but use the plugin instead.

Reply:
makes sense :)!